src/helpers.js

```diff
@@ -1,6 +1,22 @@
-// name export functions here that don't || shouldn't belong in a component.
+// import bars from '../jacobogart.js';
```

Comment:
We definitely need to check with the instructors on having this file!!

Reply:
We can definitely check! And make changes if needed.